examples/foundational/07g-interruptible-openai.py

```diff
@@ -51,6 +50,8 @@ async def main():
             ),
         )
 
+        stt = OpenAISTTService(api_key=os.getenv("OPENAI_API_KEY"), model="whisper-1")
```

Comment:
we should set `transcription_enabled=False` since we are using OpenAI's STT now.

Reply:
And we should enable audio passthrough `vad_audio_passthrough=True` so the audio can reach the STT.